weepickle/test/src/com/rallyhealth/weepickle/v1/MacroTests.scala

```diff
@@ -232,6 +232,11 @@ object MacroTests extends TestSuite {
           FromJson("""{"i":123, "j":false, "k":"haha", "s":"kk", "l":true, "z":[1, 2, 3]}""").transform(ToScala[ADTb])
         assert(r2 == ADTb(123, "kk"))
       }
+      test("ignoreAnnotatedFieldsWhenSerializing") {
+        import Annotated.D
+        val w = FromScala(D("a", Some("b"), "c")).transform(ToJson.string)
+        assert(w == """{"a":"a","c":"c"}""")
+      }
```

Comment:
Test `FromJson` too. If `b` was a `String` rather than an `Option[String]` (which has an assumed default of None), how would that work? Seems like a potentially problematic asymmetry. Perhaps we should require that all @dropAlways fields have a default (assumed or explicit) to avoid the issue?

Reply:
Right, definitely thinking along the same vein about requiring a default. https://github.com/rallyhealth/weePickle/pull/31#discussion_r444685602